build.gradle

```diff
@@ -52,7 +52,8 @@ gitHooks {
     hooksDirectory = file('.git/hooks')
     gradleCommand = './gradlew'
     hooks = [
-        'pre-push': 'clean check'
+        // Git hooks run in a bash environment regardless of OS.
+        'pre-push': 'clean check && .github/run-checks.sh'
```

Comment:
Similar to https://github.com/AY2021S1-CS2103T-W16-3/tp/pull/7#pullrequestreview-492595656, Cygwin messes up on CRLF line endings in the scripts. Not a major problem; the hooks will work as long as the scripts are changed to LF line endings.